pl_examples/ipu_examples/mnist.py

```diff
@@ -79,11 +81,20 @@ def configure_optimizers(self):
 
 
 if __name__ == '__main__':
-    dm = MNISTDataModule(batch_size=32)
+    dm = MNISTDataModule(batch_size=64)
 
     model = LitClassifier()
+    inference_opts = poptorch.Options()
+    inference_opts.replicationFactor(4)
+    inference_opts.deviceIterations(2)
+    inference_opts.Training.gradientAccumulation(1)
```

Comment:
Probably shouldn't set this. It's harmless but people might think you need to do this if it's in the example

Reply:
thanks! yep will revert all the changes here, just using it to ensure manual plugin support works